debugtools/DDR_VM/src/com/ibm/j9ddr/StructureTypeManager.java

```diff
@@ -242,12 +242,12 @@ public int getType(String type)
 		}
 		
 		// SRP?
-		if(type.startsWith("J9SRP")) {
+		if (type.equals("J9SRP") || type.startsWith("J9SRP(")) {
```

Comment:
Please explain the reason for this change and the next.

Reply:
We want to match types like `J9SRP` or `J9SRP(UDATA)` but not `J9SRP*` or `J9SRPHashTable`. I didn't encounter any troublesome types that start with `J9WSRP`, but it seemed prudent to change that test similarly.

Reply:
Seems the explanation should be a comment in the code.

Reply:
Agreed.